### DISCUSSION TITLE
Alpine Expression Error: ToastComponent is not defined

### DISCUSSION THREAD

**Discussion by kennethsantianez** [usernotnull/tall-toasts#113] [Q&A] [answered]
Hi @usernotnull , I tried to install the package on a fresh install of Laravel using Laravel Breeze starter kit. I also imported ToastComponent  in my `app.js`, already execute `php artisan optimize:clear`,but still the error persists. 

![image](https://user-images.githubusercontent.com/62993231/154969169-00232c01-7762-404e-a266-b53562351c1f.png)

-----------------
Laravel v9.1.0
Laravel Breeze v1.8
TailwindCSS v3.0.23
AlpineJS v3.4.2
Livewire v2.10

**Comment by usernotnull** [comment, marked as answer]
Hey @kennethsantianez,
You're getting this error because the component has not been initialized.

First, we need to make sure that app.js has the correct code: https://github.com/usernotnull/tall-toasts#option-1-npm-module

THEN, make sure you rebuild the frontend by calling `npm run dev`. (I'm guessing here is where you may have skipped).

The above steps should be enough. As a last resort, you could try option 2 (removing the entries you added in app.js), see https://github.com/usernotnull/tall-toasts#option-2-script-tag

**Reply by kennethsantianez** [reply]
The code in my app.js is the same as yours. Tried `npm run dev` and `npm run watch`, but still the same error.

![image](https://user-images.githubusercontent.com/62993231/154972512-4bedc888-8738-4c55-a0b8-6a555603c959.png)

Tried the option 2 and now it works! Thanks
